system-test/bigtable.ts

```diff
@@ -698,6 +698,25 @@ describe('Bigtable', () => {
         await TABLE.insert(rows);
       });
 
+      it('should include metadata in error sent back', async () => {
+        const rows = [
+          {
+            key: 'gwashington',
+            data: {
+              nonExistentColumnFamily: {
+                jadams: 1,
+              },
+            },
+          },
+        ];
+        try {
+          await TABLE.insert(rows);
+          assert.fail('Inserts should not work on non-existent column family.');
+        } catch (e: any) {
+          assert(e.metadata);
```

Comment:
I believe it's better to try and match the exact expected values of this metadata object here rather than just a blank non-falsy assertion.

Reply:
Added a check for two of the metadata properties that don't change.